python/tvm/relax/transform/legalize_ops/common.py

```diff
@@ -90,7 +90,7 @@ def _call_topi_without_attr(te_func: TEFunc, primfunc_name: Optional[str] = None
 _LEGALIZE_ATTR_NAME = "FLegalize"
 
 
-def register_legalize(op_name: str, legal_func: LegalizeFunc = None):
+def register_legalize(op_name: str, legal_func: LegalizeFunc = None, level: int = 10):
```

Comment:
Is there any necessity for `level` in this PR? 
Otherwise, I would prefer not introducing `level` yet and keep the legalizer dedicated to the reference implementation.  
If we want to lower to different implementation depending on the context, we can introduce target-specialization pass. In this way, I believe we can make the lowering process composable.

Reply:
I introduced that change for testing purpose. reverted it.